docs/testpmd.md

```diff
@@ -0,0 +1,250 @@
+# TestPMD 
+
+[TestPMD](https://doc.dpdk.org/guides/testpmd_app_ug/index.html) is anapplication  used to test the DPDK in a packet forwarding mode, here this application run inside a pod with SRIOV VF.
+
+## Pre-requisites
+
+This workload assumes a healthy cluster is runing with SR-IOV and Performance Add-On operators. 
+
+* [SR-IOV](https://docs.openshift.com/container-platform/4.6/networking/hardware_networks/installing-sriov-operator.html) operator is required to create/assign VFs to the pods
+* [PAO](https://docs.openshift.com/container-platform/4.6/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html) is required to prepare the worker node with Hugepages for DPDK application and isolate CPUs for workloads. 
+
+### Sample SR-IOV configuration
+
+These are used to create VFs and SRIOV networks, interface might vary
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: testpmd-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  nicSelector:
+    pfNames:
+     - ens2f1
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  numVfs: 20
+  resourceName: intelnics
+```
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: testpmd-sriov-network
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: |
+    {
+      "type": "host-local",
+      "subnet": "10.57.1.0/24",
+      "rangeStart": "10.57.1.100",
+      "rangeEnd": "10.57.1.200",
+      "routes": [{
+        "dst": "0.0.0.0/0"
+      }],
+      "gateway": "10.57.1.1"
+    }
+  spoofChk: "on"
+  trust: "on"
+  resourceName: intelnics
+  networkNamespace: my-ripsaw
+
+```
+
+### Sample PAO configuration
+
+The CPU and socket information might vary, this is defined based on the recommendation [here](https://docs.google.com/document/d/1wlT8z-QNzibgmNZnw5M3QNbwBcE0bJEcjWwh_BoA1Ig/edit?usp=sharing)
```

Comment:
Doubtfully this doc is public, just remove it. 

`The CPU and socket information might vary, use the below as an example`